W-12651574-use-maven-in-studio-CP (#467)
mulesoft/docs-studio: modules/ROOT/pages/configure-studio-to-use-your-own-maven.adoc

= Configure Studio to Use Your Own Maven
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

Anypoint Studio comes bundled with Maven but you can use Studio with an external Maven installation as well.

> . Navigate to Studio preferences:

Reviewer: Make sure this is consistent with "open studio and go to preferences" steps in other topics.

Author: Great catch! Thanks!

> * In macOS, go to *Anypoint Studio > Settings…*.

Reviewer: ```suggestion
* In macOS, go to *Anypoint Studio > Settings*.
```

Reviewer: (No need to include the punctuation in UI elements.)

Reviewer 2: Even when it is part of the UI? .... in particular this is a macOS managed entry... in fact with macOS Ventura it changed from `Preferences...` to `Settings...` in all the app (not just Studio).

<img width="225" alt="image" src="https://user-images.githubusercontent.com/1738654/224369010-ea9c78a7-bc13-4fbe-ba6b-4cbeec0c4c27.png">

Reviewer: Yep. That's our style.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-UIelements

> . In Windows, go to *Anypoint Studio > Preferences*.

Reviewer 2: the 2 are not indented one level

Author: You're right! Thanks!

. In the Preferences panel, go to *Anypoint Studio > Maven*.

> . Uncheck the *Use embedded Maven Installation* option.

Reviewer: ```suggestion
. Uncheck the *Use embedded Maven installation* option.
```

> . Uncheck the *Use embedded Maven Installation* option.

Reviewer: ```suggestion
. Deselect *Use embedded Maven Installation*.
```

Reviewer: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22select,%22%22choose%22

Author: OK! Thanks!

. Deselect *Use embedded Maven Installation*.

> . Click *Browse*, next to the *Maven installation home directory* field.

Reviewer: ```suggestion
. Click *Browse* next to the *Maven installation home directory* field.
```

Author: OK! Thanks!

> . Click on the *Browse* button next to the *Maven installation home directory* field.

Reviewer: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22click%22

Reviewer: Click *Browse*

Author: OK! Thanks!

> . Look for and select your Maven root installation path.

Reviewer: ```suggestion
. Select your Maven root installation path and click *Open*.
```

Author: OK! Thanks!

> . Click *Open*.

Reviewer: Combine with line 14

Author: OK! Thanks!

> . Click on the *Test Maven Configuration* button to verify your Maven configuration in Studio is correct.

Reviewer: Click *Test Maven Configuration*

Author: OK! Thanks!

+
image::maven-preferences.png["Maven options highlighted in the *Preferences* menu."]
+

> . Click the *Apply and Close* button.

Reviewer: Click *Apply and Close*

Author: OK! Thanks!
